packages/base/src/formbuilder/formselectors.ts

```diff
@@ -1,14 +1,14 @@
 import { LayerType, SourceType } from '@jupytergis/schema';
-import { GeoJSONSourcePropertiesForm } from './objectform/geojsonsource';
-import { GeoTiffSourcePropertiesForm } from './objectform/geotiffsource';
-import { HeatmapLayerPropertiesForm } from './objectform/heatmapLayerForm';
-import { HillshadeLayerPropertiesForm } from './objectform/hillshadeLayerForm';
-import { LayerPropertiesForm } from './objectform/layerform';
-import { PathBasedSourcePropertiesForm } from './objectform/pathbasedsource';
-import { TileSourcePropertiesForm } from './objectform/tilesourceform';
-import { VectorLayerPropertiesForm } from './objectform/vectorlayerform';
-import { WebGlLayerPropertiesForm } from './objectform/webGlLayerForm';
-import { SourcePropertiesForm } from './objectform/sourceform';
+import { GeoJSONSourcePropertiesForm } from './objectform/source';
+import { GeoTiffSourcePropertiesForm } from './objectform/source';
+import { HeatmapLayerPropertiesForm } from './objectform/layer';
+import { HillshadeLayerPropertiesForm } from './objectform/layer';
+import { LayerPropertiesForm } from './objectform/layer';
+import { PathBasedSourcePropertiesForm } from './objectform/source';
+import { TileSourcePropertiesForm } from './objectform/source';
+import { VectorLayerPropertiesForm } from './objectform/layer';
+import { WebGlLayerPropertiesForm } from './objectform/layer';
```

Comment:
These should be grouped

```suggestion
import { HeatmapLayerPropertiesForm, HillshadeLayerPropertiesForm, LayerPropertiesForm, PathBasedSourcePropertiesForm, ... } from './objectform/source';
```

Reply:
Ahh sorry, i didn't notice the dots, i'll corect quickly